src/viur/shop/modules/api.py

```diff
@@ -210,6 +210,12 @@ def cart_clear(
         cart_key = self._normalize_external_key(cart_key, "cart_key")
         ...
 
+    @exposed
+    def cart_list_current_user(
```

Comment:
I think I see your point. You want to save the client handling with the basket key.

I'm not against this, but we should check if this behavior is consistent with other methods as well. Additionally I don't like the method name, I would suggest `basket_list` since there is only one basket and the user could be a guest.